### PR TITLE
Add missing parameter names to event filter

### PR DESCRIPTION
**Pull request by hasparus**
Closes https://github.com/dethcrypto/TypeChain/issues/575

### CODE REVIEW

**Comment by SoheilAdmin** [comment]
bc1quc57lgsdhrxurmnp3hre07zfpx04jepdmcl65u

**Review by krzkaczor — APPROVED** [review]

File: packages/target-ethers-v5/test/generation.test.ts [hunk 1]
```diff
@@ -29,3 +30,122 @@ describe('Ethers generation edge cases', () => {
...
+        ],
+        name: 'summoned',
+        type: 'event',
+      },
+    ] as any) as RawAbiDefinition[]
+
+    const contract = parse(abi, 'Rarity')
+    const actual = Object.values(contract.events).map(generateEventFilters)
+
+    const expected = [
```
Comment (krzkaczor) [inline packages/target-ethers-v5/test/generation.test.ts:126]: i am not sure if we need such tests as we treat generated typings as a sort of snapshot tests.

Comment (hasparus) [reply packages/target-ethers-v5/test/generation.test.ts:126]: I added this one because it's easier to add `it.only` inside of Mocha test to work on a localized problem and the test descriptions document intention.

I can remove it if you'd prefer.

Comment (krzkaczor) [reply packages/target-ethers-v5/test/generation.test.ts:126]: I think it will easily break once we change anything. I would rather delete it as it doesn't provide any additional value to the snapshot test.
